### PR TITLE
Fix: Move the isUrlRelative function to utilities.client

### PR DESCRIPTION
**Pull request by m-murasovs**

### CODE REVIEW

File: src/utilities.client.js [hunk 1]
```diff
@@ -644,3 +644,12 @@ exports.splitFullName = function (fullName) {
     }
     return [names[0], nonEmptyNames.slice(1).join(' ')];
 };
+
+/**
+ * Perform a Regex test on a given URL to see if it is relative.
+ * @param  {String} url
+ * @return {boolean}
+ */
+export const isUrlRelative = (url) => {
```
Comment (jancurn) [inline src/utilities.client.js:653]: I don't see no unit test :)

Comment (m-murasovs) [reply src/utilities.client.js:653]: Busted! :D

File: test/markdown_renderers.js [hunk 2]
```diff
@@ -28,4 +30,22 @@ describe('apifyMarked custom renderer works', () => {
...
             <h3 id="welcome-title-id"><a href="#welcome-title-id"></a>Welcome to Apify</h3>`);
     });
+
+    it('replaces relative URLs from GitHub repos with absolute URLs in links', () => {
+        const href = './src/foo/bar';
+        const text = 'link to bar';
+        const repoUrl = `https://github.com/${repoFullName}`;
+        const renderedLink = customLinkRenderer(href, text, repoUrl, branchName);
+
+        expect(renderedLink).to.equal('<a href=https://github.com/apify/actor-test-url/tree/main/src/foo/bar rel="nofollow noreferrer noopener">link to bar</a>')
```
Comment (drobnikj) [inline test/markdown_renderers.js:40]: Missing double quotes
`href="https://github.com/apify/actor-test-url/tree/main/src/foo/bar"`

File: test/markdown_renderers.js [hunk 3]
```diff
@@ -28,4 +30,22 @@ describe('apifyMarked custom renderer works', () => {
...
+    })
+
+    it('replaces relative URLs in images from GitHub repos with absolute URLs pointing to raw files', () => {
+        const href = './src/foo/bar.jpeg'
+        const text = 'link to bar.jpeg';
+        const repoUrl = `https://gitlab.com/${repoFullName}`;
+        const renderedLink = customImageRenderer(href, text, repoUrl, branchName);
+
+        expect(renderedLink).to.equal(`<img src=https://gitlab.com/apify/actor-test-url/-/raw/main/src/foo/bar.jpeg alt=link to bar.jpeg />`)
```
Comment (drobnikj) [inline test/markdown_renderers.js:49]: Same here

File: test/markdown_renderers.js (hunk 2 above)
Comment (drobnikj) [inline test/markdown_renderers.js:40]: href=https://github.com/apify/actor-test-url/tree/main/src/foo/bar

File: test/markdown_renderers.js (hunk 3 above)
Comment (drobnikj) [inline test/markdown_renderers.js:49]: href=https://github.com/apify/actor-test-url/tree/main/src/foo/bar

File: test/markdown_renderers.js [hunk 4]
```diff
@@ -28,4 +30,22 @@ describe('apifyMarked custom renderer works', () => {
         expect(renderedTitle).to.equal(`
             <h3 id="welcome-title-id"><a href="#welcome-title-id"></a>Welcome to Apify</h3>`);
     });
+
+    it('replaces relative URLs from GitHub repos with absolute URLs in links', () => {
```
Comment (drobnikj) [inline test/markdown_renderers.js:34]: Maybe add one test when the URL is an absolute link already, it has to stay the same.

**Review by drobnikj — CHANGES_REQUESTED** [review]

File: test/markdown_renderers.js (hunk 2 above)
Comment (m-murasovs) [reply test/markdown_renderers.js:40]: Sorry, what do you mean with this one?

File: test/markdown_renderers.js (hunk 3 above)
Comment (m-murasovs) [reply test/markdown_renderers.js:49]: Here, I'm testing if the gitlab conversion works, should it be gihub instead?

**Review by drobnikj — APPROVED** [review]

**Pull request merged**